media/js/base/mozilla-fxa-form.js

```diff
@@ -49,21 +48,39 @@ if (typeof window.Mozilla === 'undefined') {
 
             fxaSubmitButton.disabled = false;
 
-            var destURL = fxaForm.getAttribute('action') + 'metrics-flow';
+            // track external UTM referrals
+            var urlParams = new window._SearchParams().utmParamsFxA();
```

Comment:
The `utmParamsFxA()` function here seems to have been written some time ago, and has some strange behavior which I'm not sure we're expecting here. For example, if `utm_content` is missing from the URL then it returns the pathname (e.g. `/firefox/accounts/`) instead.

I'm not entirely convinced we should be using this function here, as it seems like an old piece of code and is no longer used anywhere on the site. I'd suggest using `utmParams()` instead, since this just returns the values without any hidden logic?